code/modules/mob/living/silicon/ai/death.dm

```diff
@@ -20,7 +20,7 @@
 	SSshuttle.autoEvac()
 
 	ShutOffDoomsdayDevice()
-
+	malf_picker.remove_malf_verbs() 

```

Comment:
`malf_picker` is not going to be there for a regular AI, so it will try to call the proc of a datum that doesn't exist. Needs to check for its presence first, with 

```
if(malf_picker)
    malf_picker.remove_malf_verbs()
```